hasjob/views/helper.py

```diff
@@ -599,3 +599,9 @@ def usessl(url):
     if url.startswith('http:'):  # http://www.example.com
         url = 'https:' + url[5:]
     return url
+
+
+@app.context_processor
+def for_filter_fields():
```

Comment:
Context processor allows job_locations, job_type_choices, job_category_choices available in all the templates.
http://flask.pocoo.org/docs/0.10/templating/